setup.py

```diff
@@ -132,23 +132,20 @@ def lib_ext_kwargs(pc, prefix_env_var, lib_name, lib_pkg_name, pc_version, lib_s
             f"or ensure {lib_pkg_name}.pc is in PKG_CONFIG_PATH."
         )
 
-    crypto_extra_objects = []
     if is_win32:
         crypto_ext_lib = lib_ext_kwargs(pc, "BORG_OPENSSL_PREFIX", "libcrypto", "libcrypto", ">=1.1.1", lib_subdir="")
     elif is_openbsd:
         # Use openssl (not libressl) because we need AES-OCB via EVP api. Link
         # it statically to avoid conflicting with shared libcrypto from the base
-        # OS pulled in via dependencies.
-        crypto_ext_lib = {"include_dirs": ["/usr/local/include/eopenssl30"]}
-        crypto_extra_objects += ["/usr/local/lib/eopenssl30/libcrypto.a"]
+        # OS pulled in via dependencies. Make crypto paths configurable.
+        include_dirs = [os.environ.get("BORG_OPENSSL_INC", "/usr/local/include/eopenssl30")]
```

Comment:
about the default:

what's beneficial here to most users?

i can adapt the vagrantfile so it'll use 3.0 for there.

Reply:
Most OpenBSD users use a version of OpenBSD that doesn't have OpenSSL 3.0.

Reply:
I'll run tests on my vagrant box now.

What I meant with beneficial for most users is that there could be a default of eopenssl32 (or whatever), so that it works for most users and I would just use the env vars to make it fit the old vagrant box. of course, if there is a different openssl version used for every openbsd version, there might be no clear choice.

Reply:
due to the empty defaults, guess it currently works for nobody "out of the box" without setting the env vars.

Reply:
it worked on my vagrant box.

Reply:
> due to the empty defaults, guess it currently works for nobody "out of the box" without setting the env vars.


I set some sane defaults so that users, who run OpenBSD current, can do without setting env vars.